experimental/QuadFormAndIsom/src/enumeration.jl

```diff
@@ -0,0 +1,952 @@
+##################################################################################
+#
+# This is an import to Oscar of the methods written following the paper [BH23] on
+# "Finite subgroups of automorphisms of K3 surfaces".
+#
+##################################################################################
+
+##################################################################################
+#
+# Admissible triples
+#
+##################################################################################
+
+# The tuples in output are pairs of positive integers!
+function _tuples_divisors(d::T) where T <: Hecke.IntegerUnion
```

Comment:
```suggestion
function _tuples_divisors(d::T) where T <: IntegerUnion
```
at other places as well